<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XRtmpStreamerClass</class>
 <widget class="QWidget" name="XRtmpStreamerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>911</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XRtmpStreamer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: rgb(255, 255, 255);
	font: 75 12pt &quot;黑体&quot;;
}
QWidget
{
	background-color: rgb(53, 53, 53);
}

QPushButton:hover
{ 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(120, 120, 120, 255), stop:0.495 rgba(39, 39, 39, 255), stop:0.505 rgba(39,39, 39, 255), stop:1 rgba(120, 120,120, 255));
 border: none;
border-radius:15px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }

QPushButton:!hover
 { 
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(39, 39, 39, 255), stop:0.495 rgba(120, 120, 120, 255), stop:0.505 rgba(120,120, 120, 255), stop:1 rgba(39, 39,39, 255));
 border: none;
border-radius:15px;
color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
 }

QLineEdit
{
border-radius:10px;
background-color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
}

QComboBox
{
background-color: rgb(255, 255, 255);
font: 75 12pt &quot;黑体&quot;;
}</string>
  </property>
  <widget class="QLineEdit" name="outUrl">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>rtmp://192.168.175.130/live/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>推流RTMP URL</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>输入的URL或0,1</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inUrl">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>140</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>美颜级别：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="faceLevel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>一级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>二级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>三级</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="xrtmpstreamer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>XRtmpStreamerClass</receiver>
   <slot>startPushStream()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>991</x>
     <y>330</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startPushStream()</slot>
 </slots>
</ui>
